sparkle/src/components/AssistantPreview.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { SyntheticEvent, useState } from "react";
 
-import { Button, CardButton, PlayIcon } from "@sparkle/_index";
+import { Button, CardButton, MoreIcon, PlayIcon } from "@sparkle/_index";
 import { Avatar } from "@sparkle/components/Avatar";
 import { classNames } from "@sparkle/lib/utils";
 
-type AssistantPreviewVariant = "item" | "list" | "gallery";
+type AssistantPreviewVariant = "item" | "list" | "gallery" | "minimalGallery";
```

Comment:
nit: I think variant are more snake_case than camel_case and I would prefix with gallery instead of minimal. Maybe `gallery_minimal` or `minimal` ?

Reply:
I'm not an expert of design system best practice but shouldn't this be a size on the variant `gallery` ?

cc @Duncid

Reply:
1. Re: variant of gallery, it's probably a personal opinion but I think it carries enough differences with `gallery` to be a different variant (action button, responsive behavior). We'd end up with conditionals/ternary operators everywhere in the variant that would make it difficult to read.

2. Re: naming/casing, I don't have a strong opinion on this. A 1 minute search shows that most design systems 1/ seem to almost always stick to a single word for variant, 2/ use camelCase otherwize. Let's do `minimal` @Duncid ?